src/components/StaticAppBar/StaticAppBar.react.js

```diff
@@ -230,7 +290,14 @@ class StaticAppBar extends Component {
           .addClass('nav-down');
       }
       lastScrollTop = st;
-    }
+    };
+
+    setInterval(() => {
+      if (didScroll) {
+        hasScrolled();
+        didScroll = false;
+      }
+    }, 500);
 
     // Return menu items for the hamburger menu
     Logged = props => (
```

Comment:
Change the function name to `renderMenuItems`

Reply:
By changing the name, its not working.